ui/js/reactrevisions.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+// via pushlogRevisionsClone.html
+var pushlogRevisionComponent = React.createClass({
```

Comment:
Seems like this could have a better name.  Maybe ``moreRevisionsLinkComponent``?  :)  I'm not really sure the right name, but this might give a better indication of its purpose.